packages/shopify/composables/src/composables/useCart/index.ts

```diff
@@ -0,0 +1,88 @@
+/* istanbul ignore file */
+
+import { useCartFactory, UseCartFactoryParams } from '@vue-storefront/core';
+import { ref, Ref } from '@vue/composition-api';
+import { addToCart, removeCart, updateCart } from '@vue-storefront/shopify-api';
+import { Cart, CartItem, Product } from '@vue-storefront/shopify-api/src/types';
+import { Coupon } from '../../types';
+import loadCurrentCart from './loadCurrentCart';
+import Cookies from 'js-cookie';
+
+export const cart: Ref<Cart> = ref(null);
+
+// @todo: implement cart
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const fetchCartLineItem = ({ currentCart, product }) => {
+  const currentLineItem = currentCart.lineItems.find(lineItem => lineItem.id === product.id);
+  return currentLineItem;
+};
+
+const params: UseCartFactoryParams<Cart, CartItem, Product, Coupon> = {
+  loadCart: async () => {
+    const cartParams = {
+      checkoutId: Cookies.get('cart_id') || '',
+      id: Cookies.get('cart_id') || ''
+    };
+    const currentCart = await loadCurrentCart(cartParams);
+    cart.value = currentCart;
```

Comment:
please dont mutate the `cart` object (applies to all mutations in this file). It happens inside the factory based on value returned from this fn